astarte/device/device_mqtt.py

```diff
@@ -458,7 +466,14 @@ def __on_disconnect(self, _client, _userdata, rc):
             self.__mqtt_client.loop_stop()
         # Else check certificate expiration and try reconnection
         # TODO: check for MQTT_ERR_TLS when Paho correctly returns it
```

Comment:
Is it an old comment? Perhaps you can remove it

Reply:
Yes, looking at the sources of paho for Python it appears that `MQTT_ERR_TLS` is not used.